components/blocks/VerticalCards.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link';
+import React from 'react';
+import css from 'styled-jsx/css';
```

Comment:
I think we should stop using this package. 

Can you use `StyledComponents` or Tailwind instead

Reply:
✅